### DISCUSSION TITLE
How can I remove items from the navigation menu in the admin UI?

### DISCUSSION THREAD

**Discussion by JordenLCH** [vendurehq/vendure#2368] [Q&A] [answered]
We are current not using the 'Collections' feature. Could you please guide me on how to remove the 'Collections' tab from the side navigation menu?

I have reviewed the documentation and discovered a [method](https://docs.vendure.io/plugins/extending-the-admin-ui/modifying-navigation-items/#overriding-existing-nav-items) to customise it, but unfortunately, it lacks specific details. Additionally, I stumbled upon a related [issue](https://github.com/vendure-ecommerce/vendure/issues/562), suggesting that this functionality has been implemented. However, it appears that there are no instructions in the documentation on how to remove it.

I would greatly appreciate it if someone could provide guidance on achieving this. Even a simple code snippet would be immensely helpful.

**Comment by michaelbromley** [comment, marked as answer]
Hi,

You are right, the current docs lack examples to illustrate. I'm currently upgrading all the docs but didn't get round to that part yet.

Here's how it would work:

```ts
import { NgModule } from '@angular/core';
import { SharedModule, addNavMenuItem} from '@vendure/admin-ui/core';

@NgModule({
  imports: [SharedModule],
  providers: [
    addNavMenuItem({
      id: 'collections',  // <-- we will override the "collections" menu item
      label: 'Collections',
      routerLink: ['/catalog', 'collections'],
      requiresPermission: '__disable__' // <- we use an invalid permission which ensures it is hidden from all users
    },
    'catalog'),
  ],
})
export class MyUiExtensionModule {}
```

**Reply by JordenLCH** [reply]
Thank you for your prompt response!
